packages/svelte-undp-design/src/lib/Header/Header.svelte

```diff
@@ -10,7 +10,7 @@
 	export let showProgressBar = false;
 	export let isPositionFixed = true;
 	export let links: HeaderLink[] = [];
-
+	export let progressBarSize: 'small' | 'normal' = 'small';
```

Comment:
can you use same name with class name?

for example, options can be
xsmall, small, normal, medium, large